flytekit/clis/sdk_in_container/utils.py

```diff
@@ -79,22 +84,66 @@ def pretty_print_grpc_error(e: grpc.RpcError):
     return
 
 
-def pretty_print_traceback(e):
+def remove_unwanted_traceback_frames(
+    tb: types.TracebackType, unwanted_module_names: typing.List[str]
+) -> types.TracebackType:
     """
-    This method will print the Traceback of an error.
+    Custom function to remove certain frames from the traceback.
     """
-    if e.__traceback__:
-        stack_list = traceback.format_list(traceback.extract_tb(e.__traceback__))
-        click.secho("Traceback:", fg="red")
-        for i in stack_list:
-            click.secho(f"{i}", fg="red")
+    frames = []
+    while tb is not None:
+        frame = tb.tb_frame
+        frame_info = (frame.f_code.co_filename, frame.f_code.co_name, frame.f_lineno)
+        if not any(module_name in frame_info[0] for module_name in unwanted_module_names):
+            frames.append((frame, tb.tb_lasti, tb.tb_lineno))
+        tb = tb.tb_next
+
+    # Recreate the traceback without unwanted frames
+    tb_next = None
+    for frame, tb_lasti, tb_lineno in reversed(frames):
+        tb_next = types.TracebackType(tb_next, frame, tb_lasti, tb_lineno)
 
+    return tb_next
 
-def pretty_print_exception(e: Exception):
+
+def pretty_print_traceback(e: Exception, verbosity: int = 1):
+    """
+    This method will print the Traceback of an error.
+    Print the traceback in a nice formatted way if verbose is set to True.
+    """
+    console = Console()
+    tb = e.__cause__.__traceback__ if e.__cause__ else e.__traceback__
+
+    if verbosity == 0:
+        console.print(Traceback.from_exception(type(e), e, None))
+    elif verbosity == 1:
```

Comment:
`>= 1` right?

Reply:
If verbosity is greater than 1, it will fall to `else`. 
```python
console.print(Traceback.from_exception(type(e), e, tb))
```

Reply:
nvm, I added `elif verbosity >= 2:`